tests/test_db.py

```diff
@@ -38,3 +38,59 @@ def test_value_to_singer_value(self):
         self.assertEqual(db.selected_value_to_singer_value_impl(True, 'boolean'), True)
         self.assertEqual(db.selected_value_to_singer_value_impl(0, 'boolean'), False)
         self.assertEqual(db.selected_value_to_singer_value_impl(False, 'boolean'), False)
+
+
+    def test_prepare_columns_sql(self):
+        self.assertEqual(' "my_column" ', db.prepare_columns_sql('my_column'))
+
+    def test_prepare_columns_for_select_sql_with_timestamp_ntz_column(self):
+        self.assertEqual(
+            'CASE WHEN  "my_column"  < \'0001-01-01 00:00:00.000000\' OR '
+            ' "my_column"  > \'9999-12-31 23:59:59.999999\' THEN \'9999-12-31 23:59:59.999\' '
```

Comment:
Same as above, we need to have the dates in milliseconds otherwise dates like `9999-12-31 23:59:59.999123` or `0000-01-01 00:00:00.000123` not converted to the fallback date in milliseconds.

Reply:
btw, `0000-01-01 00:00:00.000123` is not even parsable in Python, will throw year 0 out of range, so it will use the fallback date.

Reply:
Updated in https://github.com/transferwise/pipelinewise-tap-postgres/pull/62/commits/586aba77aaf115604a5565a81a897e7bc05f5624

Reply:
thanks 👍